suite-native/accounts/src/components/AccountListPlaceholder.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Box, Pictogram } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+const PLACEHOLDER_HEIGHT = 380;
```

Comment:
Have you also tested this on iPad?

Reply:
Same as on any other device. There is a search bar and a placeholder with a height of 380 right underneath it.

Reply:
Just wondering whether the placeholder component itself shouldn't be centered horizontally in case there are no accounts to avoid bunch of whitespace underneath.

Reply:
I believe it should not, because this placeholder is shown while you type in searchbar. So the opened keyboard could overlap it if it would be centered.